src/cogs/rules.py

```diff
@@ -37,6 +47,57 @@ async def snipe(self, ctx):
         content = message.content
         return await ctx.send(f"**{discord.utils.escape_markdown(discord.utils.escape_mentions(user))}** said on {ts} UTC:\n{content}")
 
+    async def _get_dm_channel(self, member):
+        dm_channel = member.dm_channel
+        if dm_channel == None:
+            dm_channel = await member.create_dm()
+        return dm_channel
```

Comment:
```suggestion
        return member.dm_channel or await member.create_dm()
```

You don't have to create a dm channel tho, discord.py internally already does this 🤷